app/helpers/application_helper/button/vm_reconfigure_multiple.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationHelper::Button::VmReconfigureMultiple < ApplicationHelper::Button::Basic
+  def visible?
+    role_allows?(:identifier => 'vm_reconfigure_all', :any => true)
```

Comment:
same here `should this be `role_allows?(:feature => 'vm_reconfigure_all', :any => true) && @record.reconfigurable?``